src/sage/graphs/graph.py

```diff
@@ -4989,6 +4999,39 @@ def minor(self, H, solver=None, verbose=0, *, integrality_tolerance=1e-3):
             Traceback (most recent call last):
             ...
             ValueError: This graph has no minor isomorphic to H !
+
+        Trying to find an induced minor for a graph with a C6 cycle::
+
+            sage: g = graphs.CycleGraph(6)              # Create a graph with 6 vertices forming a C6 cycle
+            sage: for i in random.randint(10, 30):
+            ....:     g.add_edge(random.randint(0, 5), i)
+            sage: h = graphs.CycleGraph(5)               # Create a graph with 5 vertices forming a C5 cycle
+            sage: L = g.minor(h, induced=True)                                           
+            sage: gg = g.subgraph(flatten(L.values(), max_level = 1))              
+            sage: _ = [gg.merge_vertices(l) for l in L.values() if len(l)>1]        
+            sage: gg.is_isomorphic(h)
+            True
+
+        TESTS::
+
+            sage: g = Graph()
+            sage: g.add_edges([(0, 1), (0, 2), (1, 2), (2, 3), (3, 4), (3, 5), (4, 5), (6, 5)])
+            sage: h = Graph()
+            sage: h.add_edges([(9, 10), (9, 11), (9, 12), (9, 13)])
+            sage: l = g.minor(h, induced=True)
```

Comment:
You could add a test with `g.minor(h, induced=False)` to show the impact of this parameter.